demo/app/components/demo-map/demo-map.component.html

```diff
@@ -22,4 +22,5 @@
     <!--<arc-layer #layer></arc-layer>-->
     <!--<draw-on-map-layer></draw-on-map-layer>-->
     <!--<polygon-layer></polygon-layer>-->
+    <keyboard-control-layer *ngIf="appSettingsService.keyboardControl"></keyboard-control-layer>
```

Comment:
Do you have another idea? because I need the `KeyboardControlService`, which is a provider that created on `acMap` component. 
Is there another way to access the map-specific `KeyboardControlService`? trying to inject it into `demo-map` throws an error of missing provider.